subsys/suit/provisioning/soc/Kconfig.nrf54h20

```diff
@@ -142,7 +142,7 @@ rsource "Kconfig.template.manifest_config"
 
 config SUIT_MPI_RAD_AREA_PATH
 	string "Path to the file with all radio domain MPI configurations"
-	default "suit_mpi_radiocore_merged.hex"
+	default "suit_mpi_radio_merged.hex"
```

Comment:
From the nrfutil docs: they propose to use `Network` for the `--core` argument. I think we may keep the `radio` here and get back to this once the command to provision the device will appear.